_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -722,6 +744,18 @@ CloudTrail logs from the other accounts to this one S3 bucket. Also, make sure t
 give a small number of trusted admins access to the KMS master key and the S3 bucket. You may also want to send the
 logs to CloudWatch Logs a second way to store/view audit logs.
 
+=== AWS Config
+
+You'll want to enable AWS Config in all available regions in each of your AWS accounts so that you can track AWS
```

Comment:
The fact that you enable Config in every region is probably non-obvious and should be called out more. E.g., "You'll want to enable AWS Config in every one of your AWS accounts, plus any AWS Config Rules that are relevant to your company, so that you can track AWS resource configuration changes over time and enforce company policy. Note that you should enable AWS Config _in every AWS region_, and not just the region(s) you're using for the rest of your infrastructure That way, if an employee (perhaps accidentally), or even worse, an attacker, runs something in a region you don't typically use, it'll still be recorded in AWS Config, and your rules will still be enforced."

Same thing applies to the GuardDuty section below!

Reply:
Ok I've updated both.